scheduler/src/cook/mesos/api.clj

```diff
@@ -659,19 +648,98 @@
                      :body "Internal server error."})) )))
 
 
+
+(defn get-shares
```

Comment:
I realize I should have been more clear. 

Please call this one get-resource-types (and make the endpoint /resources/) and keep the user specific ones as share.